woocommerce/google-listings-and-ads
# Attribute Mapping - Sync Frontend

### Changes proposed in this Pull Request:

Closes part of #1648 .

This PR implements the Frontend par of https://github.com/woocommerce/google-listings-and-ads/pull/1731

That is to Poll the sync API endpoint and update the message in accordance. 

### Screenshots:

https://user-images.githubusercontent.com/5908855/196202508-c0345216-b5a1-4f93-9ef6-92dcd017cc6f.mov

### Detailed test instructions:


1. Go to Attribute Mapping. See the message "Your products are synced with your rules"
2. Create a new rule and see how the message changes to "Your products are being synced with your rules" after a few seconds.
3. Without closing the window.  Go to WooCommerce - Status - Scheduler in another window an look for `gla/jobs/update_all_products/create_batch`. Scheduled for 30 minutes. Run it manually. 
4. Go back to the other window in Attribute mapping and see how after a few seconds it updates back to "Your products are synced with your rules"
